htdocs/comm/propal/list.php

```diff
@@ -2014,9 +2015,11 @@
 			}
 			// Source - input reason
 			if (!empty($arrayfields['p.fk_input_reason']['checked'])) {
-				print '<td class="tdoverflowmax125" title="'.dol_escape_htmltag($form->cache_demand_reason[$obj->fk_input_reason]['label']).'">';
 				if ($obj->fk_input_reason > 0) {
+					print '<td class="tdoverflowmax125" title="'.dol_escape_htmltag($form->cache_demand_reason[$obj->fk_input_reason]['label']).'">';
```

Comment:
The td must not be inside this if but before and /td after. This generates bad balanced td in array